frontend/components/App.tsx

```diff
@@ -254,10 +177,9 @@ function App() {
   }
 
   return (
-    // Styled Components must be injected last in order to override Material UI style: https://material-ui.com/guides/interoperability/#controlling-priority-3
```

Comment:
These comments are good to explain why things are the way they are. I would keep the better comments